src/Build/Logging/BinaryLogger/Postprocessing/IBuildEventStringsReader.cs

```diff
@@ -18,6 +18,8 @@ public interface IBuildEventStringsReader
         public event Action<StringReadEventArgs>? StringReadDone;
 
         /// <summary>
+        /// WARNING: This event is under low support and low maintenance - please use <see cref="StringReadDone"/> instead. 
+        /// 
         /// An event that allows the caller to be notified when a string is encountered in the binary log.
         /// BinaryReader passed in ctor is at the beginning of the string at this point.
         /// </summary>
```

Comment:
We have a fancy way to do this (special tags are [in Learn docs](https://learn.microsoft.com/contribute/content/markdown-reference#alerts-note-tip-important-caution-warning)):

```suggestion
        /// <summary>
        /// An event that allows the caller to be notified when a string is encountered in the binary log.
        /// BinaryReader passed in ctor is at the beginning of the string at this point.
        /// </summary>
        /// <remarks>
        ///    <format type="text/markdown"><![CDATA[
        /// ## Remarks
        /// > [!CAUTION]
        /// > Use <see cref="StringReadDone"/> instead of this method.
        /// ]]></format>
        /// </remarks>
```